From ed740c3e4bd5aaf4af375f54a7a16554db7f1c79 Mon Sep 17 00:00:00 2001
From: Paul Gortmaker <paul.gortmaker@windriver.com>
Date: Mon, 21 Feb 2011 12:01:35 -0500
Subject: [PATCH] percpu: add missing annotations to exports for !CHECKER

In 33rt, there was only one set of percpu exports.  But in
the 33->34 dev cycle, they were cloned; one for each of
CHECKER and !CHECKER.  Currently only one of the two sets
has the RT additions.  Add the same change to the 2nd set.

Signed-off-by: Paul Gortmaker <paul.gortmaker@windriver.com>
---
 include/linux/percpu-defs.h |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/include/linux/percpu-defs.h b/include/linux/percpu-defs.h
index 4f9c20d..3511936 100644
--- a/include/linux/percpu-defs.h
+++ b/include/linux/percpu-defs.h
@@ -166,9 +166,9 @@
 #define EXPORT_PER_CPU_LOCKED_SYMBOL_GPL(var) EXPORT_SYMBOL_GPL(var##_locked)
 #else
 #define EXPORT_PER_CPU_SYMBOL(var)
-#define EXPORT_PER_CPU_LOCKED_SYMBOL(var)
+#define EXPORT_PER_CPU_LOCKED_SYMBOL(var##_locked)
 #define EXPORT_PER_CPU_SYMBOL_GPL(var)
-#define EXPORT_PER_CPU_LOCKED_SYMBOL_GPL(var)
+#define EXPORT_PER_CPU_LOCKED_SYMBOL_GPL(var##_locked)
 #endif
 
 #endif /* _LINUX_PERCPU_DEFS_H */
-- 
1.7.4

